### PR TITLE
improve introduction markdown content

### PR DESCRIPTION
**Pull request by LronDC**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
Now on the introduction page, "[install Karmada](https://karmada.io/zh/docs/install)." leads to a 404 page.
This PR fixes the path right.

**Special notes for your reviewer**:

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/lgtm

**Review by rgrupesh — COMMENTED** [review]
/lgtm

File: docs/getting-started/introduction.md [hunk 1]
```diff
@@ -48,6 +48,6 @@ Karmada is a sandbox project of the [Cloud Native Computing Foundation](https://
 
 Here are some recommended next steps:
 
-- Start to [install Karmada](./install).
 - Learn Karmada's [core concepts](./getting-started/core-concept).
 - Learn Karmada's [architecture](./getting-started/architecture).
+- Start to [install Karmada](./installation/installation).
```
[the remark this answers is not in the document]
Comment (LronDC) [reply docs/getting-started/introduction.md:53]: Well, I followed the side menu's order, should I keep the original order?

side menu:
image

[the remark this answers is not in the document]
Comment (RainbowMango) [reply docs/getting-started/introduction.md:53]: right, I didn't notice that.

**Pull request merged**